q2_bowtie2/_bowtie.py

```diff
@@ -67,6 +70,8 @@ def align_paired(
             aligned_path,
             "--al-conc-gz",
             unaligned_path,
+            "-p/--threads",
```

Comment:
This is going to fail. It needs to be either `-p` or `--threads`

Reply:
Ah okay, I thought this was weird command. Didn't realize it was being used as `either, or`.